src/node.cc

```diff
@@ -176,7 +176,7 @@ bool ssl_openssl_cert_store =
 bool enable_fips_crypto = false;
 bool force_fips_crypto = false;
 # endif  // NODE_FIPS_MODE
-const char* openssl_config = nullptr;
+const char* openssl_config = secure_getenv("OPENSSL_CONF");
```

Comment:
This is not safe, modifications to the environment will invalidate the pointer.  Call `secure_getenv()` only when `openssl_config` is first needed and store a copy, not a pointer.

I see we're also mishandling `icu_data_dir` that way.  I'll file a pull request.